lessons/introduction-to-smartpy/02/02.mdx

```diff
@@ -7,9 +7,11 @@ editor:
   startingCode: |
     import smartpy as sp
 
+    # Change the comment to `Plant Contract`
```

Comment:
the previous state makes sense here, we are hand-holding user too much

Reply:
revert to previous state here

Reply:
Ok